#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_fix_segfault.dpatch 
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debian-specific changes to the global pinforc file

@DPATCH@

--- pinfo-0.6.9rc2/src/filehandling_functions.c	2008-06-04 16:10:36.021052577 +0200
+++ pinfo-0.6.9rc2/src/filehandling_functions.c	2008-06-04 16:10:48.055848333 +0200
@@ -200,7 +200,7 @@
 	}
 
 	/* if we haven't found anything, clean up and exit */
-	if (!goodHit)
+	if ( id  &&  !goodHit )
 	{
 		fclose(id);
 		id = 0;

